symfony/framework-bundle/3.3/public/index.php

```diff
@@ -21,7 +21,13 @@
     Debug::enable();
 }
 
-// Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
+if ($trustedProxies = $_SERVER['APP_TRUSTED_PROXIES'] ?? false) {
+    Request::setTrustedProxies(json_decode($trustedProxies, true), Request::HEADER_X_FORWARDED_ALL);
+}
+
+if ($trustedHosts = $_SERVER['APP_TRUSTED_HOSTS'] ?? false) {
+    Request::setTrustedHosts(json_decode($trustedHosts, true));
```

Comment:
I would suggest to move this in 
 symfony/framework-bundle/3.3/config/packages/framework.yaml now that we can json-decode env vars